PROFILING.md

```diff
@@ -0,0 +1,46 @@
+h1. Profiling linkerd-tcp
```

Comment:
Tiny nit: I forget which syntax uses `h1.`, but it's not markdown :)

![screen shot 2017-04-13 at 4 51 02 pm](https://cloud.githubusercontent.com/assets/9226/25028474/6f8da0f4-2069-11e7-863c-f2398f518cbf.png)